app/assets/stylesheets/application.css.scss

```diff
@@ -13,5 +13,18 @@
 // Customize
 @import "content";
 @import "style";
+
+// Controller
 @import "pages";
 @import "search";
+@import "settings";
+
+.flash {
+  margin: 0 10px 20px 10px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 10px 20px` instead of `0 10px 20px 10px`